yax/core/src/test/scala/doobie/syntax/catchable.scala

```diff
@@ -22,7 +24,7 @@ object catchablespec extends Specification {
     }
 
     "work on unaliased IConnection" in {
-      42.pure[ConnectionIO].map(_ + 1).attempt
+      (42.pure[ConnectionIO].map(_ + 1): ConnectionIO[Int]).attempt
```

Comment:
Let's leave this test as-is and yax it out for cats, then open a separate issue to sort this out. I know Adelbert is pretty focused on it so maybe he'll have some ideas.